### ISSUE TITLE
Move object management to Tasks.

### ISSUE THREAD

**Issue by steveperry-53** [kubernetes/website#14272]
**This is a Feature Request**

**What would you like to be added**

Move these topics to the Tasks section:

* [Managing Kubernetes Objects Using Imperative Commands](https://kubernetes.io/docs/concepts/overview/object-management-kubectl/imperative-command/)

* [Imperative Management of Kubernetes Objects Using Configuration Files](https://kubernetes.io/docs/concepts/overview/object-management-kubectl/imperative-config/)

* [Declarative Management of Kubernetes Objects Using Configuration Files](https://kubernetes.io/docs/concepts/overview/object-management-kubectl/declarative-config/)

* [Declarative Management of Kubernetes Objects Using Kustomize](https://kubernetes.io/docs/concepts/overview/object-management-kubectl/kustomization/)

I would suggest something like this:

```
Tasks
    Install Tools
    Configure Pods and Containers
    Manage Kubernetes Objects
        Manage Kubernetes Objects Using Imperative Commands
        Imperative Management of Kubernetes Objects Using Configuration Files
        Declarative Management of Kubernetes Objects Using Configuration Files
        Declarative Management of Kubernetes Objects Using Kustomize
```

Take this topic out from under **Object Management Using kubectl**, and put it under **Working with Kubernetes Objects**:

* [Kubernetes Object Management](https://kubernetes.io/docs/concepts/overview/object-management-kubectl/overview/)

**Why is this needed**
The content in these topics is not conceptual; it is how-to. So it belongs under Tasks.

**Related**

**Comment by kbhawkey** [comment]
Hi. I agree that most of the kubectl and kustomize related content should be moved, though, the last three topics contain a number of embedded concepts, not just `do task 1, followed by task 2 in my cluster ...`. I also think that the task area has grown very large so that it is hard to find topics/tasks. The other question is how to associate, group, and maintain the kubectl content in the `reference` section with the content in the kubectl-related task section.

**Comment by kbhawkey** [comment]
@steveperry-53, if it is okay with you, would you assign this `task` to me?

**Comment by steveperry-53** [comment]
@kbhawkey I understand the concerns you've raised above. So we might want to do some refactoring of content in addition to moving some of the content to Tasks. I would think that all of the conceptual content could be in one topic. And in the one conceptual topic, we could have highly visible links to the task topics.

As for your question on associating task content with reference content, I'll leave that to other members of SIG docs who are working on the whole area of ref doc generation. For example @jimangel.

**Comment by steveperry-53** [comment]
@kbhawkey It looks like you'll have to join the Kubernetes org before we can assign an issue to you. So I've assigned this to myself for now, just to hold on to it. If you'd like, go ahead and start a PR.

**Comment by kbhawkey** [comment]
Thanks, joining is on my todo list 😃 . I will start with moving the existing content and think about what information could be pulled out and remain in the concepts section.

**Comment by zacharysarah** [comment]
Closed by #14374

Thanks @kbhawkey!
/close
